docs/application/native/guides/ui/tizen-ws-shell/tzsh-softkey.md

```diff
@@ -0,0 +1,207 @@
+   main_win = elm-win_util_standard_add(name, name);
+
+   /* set up main window */
+   ...
+   evas_object_show(main_win);
+
+
+   tzsh_h tzsh;
+   tzsh_softkey_h tzsh_sk;
+   tzsh_window tz_win;
+
+   /* Get native window ID of main window */
+   tz_win = elm_win_window_id_get(main_win);
+
+   /* Create tzsh_h structure */
+   tzsh = tzsh_create(tz_win);
+
+   /* Create tzsh_softkey_h structure */
+   tzsh_sk = tzsh_softkey_create(tzsh, tz_win);
+
+   ...
+   /* Do Something Creative */
+```
+
+## Show or Hide Softkey Window
+
+To show or hide Softkey service window when your application's window is activated, call the following functions:
+
+```
+   int ret;
+
+   if (show) {
+      ret = tzsh_softkey_global_show(tzsh_sk); /* To show softkey service window */
+      if (ret != TZSH_ERROR_NONE) {
+         /* error */
+      }
+   } else {
+      ret = tzsh_softkey_global_hide(tzsh_sk); /* To hide softkey service window */
+      if (ret != TZSH_ERROR_NONE) {
+         /* error */
+      }
+   }
+```
+
+## Get Visibility Status of Softkey Window
+
+To get the current visible state of Softkey service window, call `tzsh_softkey_global_visible_get`. The Softkey service window will be visible or invisible depending on the visible state.
+
+```
+   int ret;
+   tzsh_softkey_state_visible_e state;
+
+   ret = tzsh_softkey_global_visible_state_get(tzsh_sk, &state);
+   if (ret != TZSH_ERROR_NONE) {
+      /* error */
+   }
+
+   if (state == TZSH_SOFTKEY_STATE_VISIBLE_SHOW) {
+      /* visible state */
+   } else if (state == TZSH_SOFTKEY_STATE_VISIBLE_HIDE) {
+      /* invisible state */
+   } else {
+      /* error */
+   }
+```
+
+## Set Expandable Status of Softkey Window
+
+To make the Softkey service window can expandable or not, call `tzsh_softkey_global_expand_state_set` as follows:
+
+```
+   int ret;
+
+   if (expandable) {
+      ret = tzsh_softkey_global_expand_state_set(tzsh_sk, TZSH_SOFTKEY_STATE_EXPAND_ON);
+      if (ret) {
+         /* error */
+      }
+   } else {
+      ret = tzsh_softkey_global_expand_state_set(tzsh_sk, TZSH_SOFTKEY_STATE_EXPAND_OFF);
+      if (ret) {
+         /* error */
+      }
+   }
+```
+
+## Get Expandable Status of Softkey Window
+
+To get the current expandable state of the Softkey service window, call `tzsh_softkey_global_expand_state_get`. The Softkey service window will expandable or inexpandable depend on expandable state.
+
+```
+   int ret;
+   tzsh_softkey_expand_state_e state;
+
+   ret = tzsh_softkey_global_expand_state_get(tzsh_sk, &state);
+   if (ret != TZSH_ERROR_NONE) {
+      /* error */
+   }
+
+   if (state == TZSH_SOFTKEY_STATE_EXPAND_ON) {
+      /* expand state */
+   } else if (state == TZSH_SOFTKEY_STATE_EXPAND_OFF) {
+      /* inexpand state */
+   } else {
+      /* error */
+   }
+```
+
+## Set Opacity State of Softkey Window
+
+To make the Softkey service window opaque or transparent, call `tzsh_softkey_global_opacity_state_set` as follows:
```

Comment:
How about `tzsh_softkey_global_opacity_state_set()` to make it clear that it is a function?

Reply:
Yes, I agree. This is the best way to clarify.

Reply:
Looks good to me. Modified all of functions.